cluster/gce/configure-vm.sh

```diff
@@ -559,11 +559,8 @@ if [[ -z "${is_push}" ]]; then
   set-kube-env
   [[ "${KUBERNETES_MASTER}" == "true" ]] && mount-master-pd
   create-salt-pillar
-  if [[ "${KUBERNETES_MASTER}" == "true" ]]; then
-    create-salt-master-auth
```

Comment:
> the 10.* IP of the master (the external ipv4 address)

do you mean s/external/internal there?

@roberthbailey thanks for looking into this. I didn't see how this would fix the internal vs external IP address issue, but I figured I'd prepare this essentially as a revert because my previous PR was mentioned as a possible root cause.

Would it be useful to add the `salt-set-apiserver` call to this and get it in before your #6949 lands? Any idea how long it will be? This absolutely floods the master's kubelet logs making them pretty hard to parse.